Library/Homebrew/test/support/helper/integration_command_test_case.rb

```diff
@@ -27,68 +28,42 @@ def cmd_id_from_args(args)
     "[#{test_pretty}] brew #{args_pretty}"
   end
 
-  def cmd_output(*args)
-    # 1.8-compatible way of writing def cmd_output(*args, **env)
-    env = args.last.is_a?(Hash) ? args.pop : {}
-    cmd_args = %W[
-      -W0
-      -I#{HOMEBREW_LIBRARY_PATH}/test/support/lib
-      -I#{HOMEBREW_LIBRARY_PATH}
-      -rconfig
+  def cmd_run(*args)
+    env = (args.first.is_a?(Hash) ? args.shift : {}).merge(
```

Comment:
It's more consistent with those but less with idiomatic Ruby as a whole. I don't think it's more readable so let's leave as-is, thanks.